src/dashboard/index.js

```diff
@@ -6,10 +6,9 @@ import DashboardMain from './DashboardMain'
 const { remote } = require('electron')
 const app = remote.app
 
-import i18n from 'format-message'
-i18n.setup({
-  translations: require('../../locales'),
-  locale: app.getLocale() || 'en'
+i18n.setup({	
```

Comment:
it looks like this file doesn't import i18n any more

Reply:
I'll fix it.